test/house_test.rb

```diff
@@ -0,0 +1,140 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+
+class HouseTest < Minitest::Test
+  def test_it_exists
+    house = House.new("$400000", "123 sugar lane")
+    assert_instance_of House, house
+  end
+
+  def test_it_has_a_price
+    house1 = House.new("$400000", "123 sugar lane")
+    assert_equal 400000, house1.price
+
+    house2 = House.new("250000", "124 sugar lane")
+    assert_equal 250000, house2.price
+  end
+
+  def test_it_has_an_address
+    house1 = House.new("$400000", "123 sugar lane")
+    assert_equal "123 sugar lane", house1.address
+
+    house2 = House.new("250000", "999 Other Way")
+    assert_equal "999 Other Way", house2.address
+  end
+
+  def test_it_has_rooms
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal [], house.rooms
+
+    room1 = Room.new(:bedroom, 10, '13')
+    room2 = Room.new(:bedroom, 11, '15')
+
+    house.add_room(room1)
+    house.add_room(room2)
+    assert_equal [room1, room2],house.rooms
+  end
+
+  def test_if_it_is_above_market_average
+    house1 = House.new("$600000", "123 ABC Lane")
+    assert_equal true, house1.above_market_average?
+
+    house2 = House.new("$400000", "123 sugar lane")
+    assert_equal false, house2.above_market_average?
```

Comment:
here, it's good to test two houses, since they give you different outcomes from the method you're testing. good work.